openpype/hosts/max/plugins/publish/validate_frame_range.py

```diff
@@ -27,38 +28,58 @@ class ValidateFrameRange(pyblish.api.InstancePlugin,
 
     label = "Validate Frame Range"
     order = ValidateContentsOrder
-    families = ["maxrender"]
+    families = ["camera", "maxrender",
+                "pointcache", "pointcloud",
+                "review", "redshiftproxy"]
     hosts = ["max"]
     optional = True
     actions = [RepairAction]
 
     def process(self, instance):
         if not self.is_active(instance.data):
-            self.log.info("Skipping validation...")
+            self.log.debug("Skipping Validate Frame Range...")
             return
-        context = instance.context
 
-        frame_start = int(context.data.get("frameStart"))
-        frame_end = int(context.data.get("frameEnd"))
-
-        inst_frame_start = int(instance.data.get("frameStart"))
-        inst_frame_end = int(instance.data.get("frameEnd"))
+        frame_range = get_frame_range(
+            asset_doc=instance.data["assetEntity"])
 
+        inst_frame_start = instance.data.get("frameStart")
+        inst_frame_end = instance.data.get("frameEnd")
```

Comment:
We should actually be checking `frameStart == frameStart`, `handleStart == handleStart` etc. Right?

Now technically the handles could still differ with the asset? Or does the instance not allow to specify handles?

Reply:
They have relatively different timeline or logics on frame range comparing with maya and houdini. (animation range only in Max)
Max
![image](https://github.com/ynput/OpenPype/assets/64118225/cbe85f11-354c-459e-b994-0b3b4032a11c)
Houdini
![image](https://github.com/ynput/OpenPype/assets/64118225/7fccf6fc-4221-4e18-adf7-5b4d12cc820d)
Maya
![image](https://github.com/ynput/OpenPype/assets/64118225/8508c7de-ee42-4152-a433-cce8908e9a1d)

Reply:
Sure, but preferably the frame range would be defined on the instance like in Maya - no? :) But probably best for a separate PR.

Reply:
we can have instance.data["frameStartHandle"] but there is no way to check frameStart==framestart, handlestart==handlestart unless we build some dummy data with the calculation in the collector.

Reply:
Ignore that - if we have no handles data for now I suppose we can skip checking handles for the time being.

I'd still look into [this](https://github.com/ynput/OpenPype/pull/5757#discussion_r1366917403) down the line but not for this PR. By implementing that each instance can at least also define its own frame range, etc. and also their handles, etc.